src/components/SubscriptAvatar.js

```diff
@@ -34,60 +37,76 @@ const defaultProps = {
     mainAvatar: {},
     secondaryAvatar: {},
     noMargin: false,
+    showTooltip: true,
 };
 
-function SubscriptAvatar(props) {
-    const isSmall = props.size === CONST.AVATAR_SIZE.SMALL;
-    const subscriptStyle = props.size === CONST.AVATAR_SIZE.SMALL_NORMAL ? styles.secondAvatarSubscriptSmallNormal : styles.secondAvatarSubscript;
+function SubscriptAvatar({size, backgroundColor, mainAvatar, secondaryAvatar, noMargin, showTooltip}) {
+    const isSmall = size === CONST.AVATAR_SIZE.SMALL;
+    const subscriptStyle = size === CONST.AVATAR_SIZE.SMALL_NORMAL ? styles.secondAvatarSubscriptSmallNormal : styles.secondAvatarSubscript;
     const containerStyle = isSmall ? styles.emptyAvatarSmall : styles.emptyAvatar;
     // Default the margin style to what is normal for small or normal sized avatars
     let marginStyle = isSmall ? styles.emptyAvatarMarginSmall : styles.emptyAvatarMargin;
 
     // Some views like the chat view require that there be no margins
-    if (props.noMargin) {
+    if (noMargin) {
         marginStyle = {};
     }
+
+    const mainAvatarElement = (
+        <Avatar
+            containerStyles={StyleUtils.getWidthAndHeightStyle(StyleUtils.getAvatarSize(size || CONST.AVATAR_SIZE.DEFAULT))}
+            source={mainAvatar.source}
+            size={size || CONST.AVATAR_SIZE.DEFAULT}
+            name={mainAvatar.name}
+            type={mainAvatar.type}
+            fallbackIcon={mainAvatar.fallbackIcon}
+        />
+    );
+
+    const secondaryAvatarElement = (
+        <View
+            style={[size === CONST.AVATAR_SIZE.SMALL_NORMAL ? styles.flex1 : {}, isSmall ? styles.secondAvatarSubscriptCompact : subscriptStyle]}
+            // Hover on overflowed part of icon will not work on Electron if dragArea is true
+            // https://stackoverflow.com/questions/56338939/hover-in-css-is-not-working-with-electron
+            dataSet={{dragArea: false}}
+        >
+            <Avatar
+                iconAdditionalStyles={[
+                    StyleUtils.getAvatarBorderWidth(isSmall ? CONST.AVATAR_SIZE.SMALL_SUBSCRIPT : CONST.AVATAR_SIZE.SUBSCRIPT),
+                    StyleUtils.getBorderColorStyle(backgroundColor),
+                ]}
+                source={secondaryAvatar.source}
+                size={isSmall ? CONST.AVATAR_SIZE.SMALL_SUBSCRIPT : CONST.AVATAR_SIZE.SUBSCRIPT}
+                fill={themeColors.iconSuccessFill}
+                name={secondaryAvatar.name}
+                type={secondaryAvatar.type}
+                fallbackIcon={secondaryAvatar.fallbackIcon}
+            />
+        </View>
+    );
```

Comment:
Yeah, I do not like it either, but did this way because I saw in other components and thought it was the project pattern. Ideally I would create a separate component on top of this one (not a new file), because this is being used just to conditional render the avatar with or without the tooltip wrapper. WDYT?